### DISCUSSION TITLE
Does Streamz support modin/vaex/koalas? #428

### DISCUSSION THREAD

**Discussion by enthusiastics** [blue-yonder/tsfresh#887] [Q&A] [answered]
Does Streamz support modin or vaex? Besides pandas/dask, I'm thinking of trying a few more solutions.

**Comment by nils-braun** [comment, marked as answer]
With streamz, do you refer to [this package](https://streamz.readthedocs.io/en/latest/)? If yes, I think this question is better suited for the streamz repository - this is the repository of tsfresh, a feature extraction library on time series. (From my knowledge on streamz: the lower level building blocks are framework independent, however the higher-level convenience functions are tailored to dask)
If your question was if you can use tsfresh with those libraries: the input format of tsfresh is pandas tables, however as far as I know all of the mentioned libraries have a pandas export.

**Comment by enthusiastics** [comment]
I see! Thanks so much! My question is indeed "if you can use tsfresh with those libraries".

**Reply by nils-braun** [reply]
Ok, I see. So yes, you can probably expert into pandas dataframes and feed them into tsfresh, but you will loose the parallelism that these libraries have. It is not impossible to build something better also for those libraries, similar to what we have for dask, but I am unfortunately not an expert in those tools. 

Spark is already supported, so koalas should be easier (but I have never tried)
